pytest-server-fixtures/pytest_server_fixtures/http.py

```diff
@@ -81,13 +76,12 @@ def get(self, path, as_json=False, attempts=25):
             This function will retry up to `attempts` times on connection errors, to handle 
             the server still waking up. Defaults to 25.
         """
+        e = None
```

Comment:
This was causing a lint error due to a possibly-uninitialised variable. Could be initialised to a meaningful error?